src/System.IO.Compression/tests/ZipArchive/zip_ReadTests.cs

```diff
@@ -9,53 +9,35 @@ namespace System.IO.Compression.Tests
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) // [ActiveIssue(846, PlatformID.AnyUnix)]
-            {
-                await ZipTest.IsZipSameAsDirAsync(
-                    ZipTest.zfile("unicode.zip"), ZipTest.zfolder("unicode"), ZipArchiveMode.Read);
-            }
-        }
-
-        [Fact]
-        public static async Task ReadStreaming()
+        [Theory]
+        [InlineData("normal.zip", "normal")]
+        [InlineData("fake64.zip", "small")]
+        [InlineData("empty.zip", "empty")]
+        [InlineData("appended.zip", "small")]
+        [InlineData("prepended.zip", "small")]
+        [InlineData("emptydir.zip", "emptydir")]
+        [InlineData("small.zip", "small")]
+        [InlineData("unicode.zip", "unicode")]
+        public static async Task ReadNormal(string zipFile, string zipFolder)
         {
-            //don't include large, because that means loading the whole thing in memory
-
-            await TestStreamingRead(ZipTest.zfile("normal.zip"), ZipTest.zfolder("normal"));
-            await TestStreamingRead(ZipTest.zfile("fake64.zip"), ZipTest.zfolder("small"));
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) // [ActiveIssue(846, PlatformID.AnyUnix)]
-            {
-                await TestStreamingRead(ZipTest.zfile("unicode.zip"), ZipTest.zfolder("unicode"));
-            }
-            await TestStreamingRead(ZipTest.zfile("empty.zip"), ZipTest.zfolder("empty"));
-            await TestStreamingRead(ZipTest.zfile("appended.zip"), ZipTest.zfolder("small"));
-            await TestStreamingRead(ZipTest.zfile("prepended.zip"), ZipTest.zfolder("small"));
-            await TestStreamingRead(ZipTest.zfile("emptydir.zip"), ZipTest.zfolder("emptydir"));
+            await ZipTest.IsZipSameAsDirAsync(ZipTest.zfile(zipFile), ZipTest.zfolder(zipFolder), ZipArchiveMode.Read);
         }
 
-        private static async Task TestStreamingRead(String zipFile, String directory)
+        [Theory]
+        [InlineData("normal.zip", "normal")]
+        [InlineData("fake64.zip", "small")]
+        [InlineData("empty.zip", "empty")]
+        [InlineData("appended.zip", "small")]
+        [InlineData("prepended.zip", "small")]
+        [InlineData("emptydir.zip", "emptydir")]
+        [InlineData("small.zip", "small")]
+        [InlineData("unicode.zip", "unicode")]
+        public static async Task TestStreamingRead(string zipFile, string zipFolder)
         {
-            using (var stream = await StreamHelpers.CreateTempCopyStream(zipFile))
+            using (var stream = await StreamHelpers.CreateTempCopyStream(ZipTest.zfile(zipFile)))
             {
                 Stream wrapped = new WrappedStream(stream, true, false, false, null);
-                ZipTest.IsZipSameAsDir(wrapped, directory, ZipArchiveMode.Read, false, false);
+                ZipTest.IsZipSameAsDir(wrapped, ZipTest.zfolder(zipFolder), ZipArchiveMode.Read, false, false);
```

Comment:
Nit: use named parameters to improve readability of bool parameters. (Maybe done at some point later in a separate cleanup PR)